.github/workflows/release.yml

```diff
@@ -0,0 +1,123 @@
+name: Provwasm Smart Contract Binary Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"     # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+
+jobs:
+  #  This action cleans up previously running instances of a workflow on the same branch. This accomplishes
+  #  the task of automatically cancelling CI runs on pushes to the same branch, which is a common feature in
+  #  most CI systems but currently not possible with GitHub actions.
+  cleanup_runs:
+    runs-on: ubuntu-latest
+    name: Cleanup Workflows
+    steps:
+      - uses: rokroskar/workflow-run-cleanup-action@v0.3.3
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main'"
+
+  build_init:
+    runs-on: ubuntu-latest
+    name: Build Init
+    steps:
+      - id: version
+        run: echo "::set-output name=version::${GITHUB_REF##*/}"
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+      go_version: "1.17"
+
+  build_wasm:
+    runs-on: ubuntu-latest
+    needs:
+      - build_init
+    name: Build Wasm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup go
+        uses: actions/setup-go@v2.1.5
+        with:
+          go-version: ${{ needs.build_init.outputs.go_version }}
+      - name: Install deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libleveldb-dev
```

Comment:
I bet the cosmwasm stuff doesn't need this `libleveldb-dev` package. The `provenanced` build needs it in order to enable `cleveldb` support. But I don't think wasm has anything to do with that.